tests/tpch/dask_queries.py

```diff
@@ -1106,7 +1106,7 @@ def query_20(dataset_path, fs, scale):
     ]
     res_1 = (
         res_1.groupby(["l_partkey", "l_suppkey"])["l_quantity"]
-        .sum()
+        .sum(split_out=True)
```

Comment:
Is this something we could do automatically? Why is this necessary?

Reply:
We are already shuffling but partitions are still too big after the groupby, which slows us down quite a bit. We might want to reevaluate the factor, but this doesn't really aggregate, so True is the best value here

Reply:
Yeah there is always the case where we don't reduce much at all, we could use split_out=True by default but this creates also tiny partitions in other scenarios, I don't think that we can hit every case without cardinality information

Reply:
nope this can go away